build/release.sh

```diff
@@ -40,6 +40,8 @@ if [[ $KUBE_RELEASE_RUN_TESTS =~ ^[yY]$ ]]; then
   kube::build::run_build_command make test-integration
 fi
 
+kube::build::copy_output
```

Comment:
@shashidharatd this looks reasonable to me.

Not entirely sure who is the right person to ping here for a final sanity check. 
@david-mcmahon, @jbeda could you please take a look at this or point to the right people?